src/pageObjects/BasePage.js

```diff
@@ -0,0 +1,15 @@
+import {expect} from "@playwright/test";
+import SignUp from "../components/SignUP.js";
+
+export default class BasePage {
+    constructor(page, url, waitPageLocator) {
+        this._page = page
+        this._url = url
+        this._waitPageLocator = waitPageLocator
+        this.signUp = new SignUp(page)
```

Comment:
this.signUp = new SignUp(page)

it should not be at BasePage because this popup appears only on welcome page

Reply:
fixed